PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,119 @@
+- [ ] The components are aligned with each other when possible.
+- [ ] Components are grouped based off of functionality.
+    - E.g. all components for CAN should be grouped.
+- [ ] Bypass capacitors are less than 1cm away from their respective IC's power pins.
+
+### 3D Spacing
+- [ ] Layout of components have been placed with mounting location and usage of the board in mind.
+    - Are PCBs going to be stacked on above/below this board? Are tall components placed accordingly?
+    - Are buttons and lights easily accessible and viewable?
+    - When mounted into the car, are the heights of the components and connectors accounted for?
+- [ ] I am not reading every word and I need to start this checklist over.
+    - Please uncheck everything you have checked and start over.
+- [ ] Location of connectors and wires going out of the board will prevent messy wiring.
+    - Are all the wires that are going in the same direction placed on the same edge of the board?
+
+### Components
+- [ ] Do footprints match the schematic?
+    - Make sure pins on the footprint correspond to pins in the schematic 
+    - (e.g. counterclockwise pins on footprint and alternating pins on schematic is *bad*)
+- [ ] Custom footprints have been double checked with the datasheet.
+- [ ] Footprints are consistent across the PCB: *i.e. all 0805 for Resistors/Capacitors*
+- [ ] Pin 1 of the footprint is labeled in some way. 
+- [ ] Are LED's in easy to see places? 
+- [ ] Are test points in easy to reach places?  
+- [ ] Are critical paths of switching converters as small as possible?  
+
+### Copper Layer 
+- [ ] The trace widths and trace clearances are greater than JLCPCB's minimum requirements. 
+    - [ ] Are signal traces 6mils unless provided with reasoning?   
+        - NOTE: One net can have multiple trace widths  
+- [ ] The trace lengths are as short as possible. 
+    - Can there be a more optimal route if you go to another layer? 
+- [ ] Each trace's width is capable of handling the expected current flow. 
+    - Use PCB width calculator to calculate trace width. 
+- [ ] *No sharp corners. No trace angles equal to or less than 90 degrees. 
+    - Orthogonal traces should have vias if necessary. 
+- [ ] Are edges of board surrounded by clean ground on both layers with stitching vias?  
+- [ ] Traces are in parallel with each other when possible. 
+    - E.g. traces connected between an IC and MCU can be placed in parallel with each other. 
+- [ ] There are no random trace appendages. 
+- [ ] Vias placed in copper pads are fully encompassed in the copper pads. 
+- [ ] Through-hole components do not have extraneous vias. 
+- [ ] Vias should be appropriately sized according to the connected net(s) and expected current capacity.
+
+*Not really a problem for modern manufacturing techniques but good practice and important for high speed signal integrity.
+
+### Silkscreen Layer
+- [ ] Visible text sizes are greater than JLCPCB's minimum requirements.
```

Comment:
For congested layouts, JLCPCB has been able to do 0.8 | 0.8 | 0.15 text for me without issues on the majority of my boards.